src/general/Mutations.cs

```diff
@@ -441,6 +418,21 @@ private Color RandomColour(float opaqueness)
             opaqueness);
     }
 
+    /// <summary>
+    ///   Used to determine if a newly mutated species needs to be in a different genus.
+    /// </summary>
+    /// <param name="species1">The first species. Function is not order-dependent.</param>
+    /// <param name="species2">The second species. Function is not order-dependent.</param>
+    /// <returns>True if the two species should be a new genus, false otherwise.</returns>
+    private bool NewGenus(MicrobeSpecies species1, MicrobeSpecies species2)
+    {
+        var species1UniqueOrganelles = species1.Organelles.Select(o => o.Definition).ToHashSet();
+
+        var species2UniqueOrganelles = species2.Organelles.Select(o => o.Definition).ToHashSet();
+
+        return !species1UniqueOrganelles.SetEquals(species2UniqueOrganelles);
```

Comment:
I was thinking about future configurability. And I think could this work so that set disjunction is used here and based on how many elements that has (ie. the total number of differences) would be compared against a value in Constants, that way it would be easy to tweak how different something needs to be to be a different genus. I think this approach of any organelle type changes, might result in too many new genuses starting, so this would be nice to be a bit configurable if we need to tweak this in the future.